eksconfig/add-on-node-groups.go

```diff
@@ -56,6 +56,14 @@ type AddOnNodeGroups struct {
 	ASGs map[string]ASG `json:"asgs,omitempty"`
 }
 
+type NGClusterAutoScaler struct {
+	Enable bool `json:"enable"`
+	// Created is true when the resource has been created.
+	// Used for delete operations.
+	Created         bool               `json:"created" read-only:"true"`
+	TimeFrameCreate timeutil.TimeFrame `json:"time-frame-create" read-only:"true"`
```

Comment:
Can we add another field `Image string`, and set default value.

Reply:
We are not doing this for now. Giving that option would lead into other problems as supplying wrong image that's not in GCR etc.